website/pages/de/versions.js

```diff
@@ -13,7 +13,7 @@ const Container = CompLibrary.Container;
 
 const CWD = process.cwd();
 
-const versions = require(`${CWD}/versions.json`);
+const versions = require(`${CWD}/versions.json`).filter(version => version != "12.0");
```

Comment:
Does it not automatically work this out? https://manual.avolites.de/de/versions/

Reply:
v12 was showing up but isn't translated

Reply:
Even in build?

Reply:
Yes, it didn't show up on Sebastians because he'd removed v12 from versions.json